code/marcelo/guess_lab.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+computer= random.randint(1,10)
+trys= 0
+while True:
+    computer= random.randint(1,10)
+    user= input('try to guess the right number: ')
+    user=int(user)
+    print(trys)
+    if user == computer:
+        print(f"correct number {user} matches {computer}, it took you {trys} attempts")
+        break
+    if user>computer:
+        trys += 1
+        # print(f'your number {user} is wrong. try again {computer}')
+        print(f' youre number {user} was too high. guess again')
+        print (f'you are curretly at {trys} attempts')
+    if user<computer:
+        print(f' youre number {user} was too low. guess again')
```

Comment:
if you want to use commas etc in a string, check this out
https://www.w3schools.com/python/gloss_python_escape_characters.asp